src/test/java/core/basesyntax/FileEditor.java

```diff
@@ -0,0 +1,206 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.List;
+import java.util.Scanner;
+
+import static java.nio.file.Files.getLastModifiedTime;
+
+public class FileEditor {
+    String tempTextToFile;
+    String inputFromConsole;
+    private static final DateFormat FORMAT = new SimpleDateFormat("dd MM yyyy HH:mm:ss");
+
+    public static void main(String[] args) {
+        FileEditor start = new FileEditor();
+        start.FilesEditorStart();
+    }
+
+    public void FilesEditorStart() {
+        System.out.println("To use the program, enter the command and press Enter");
+        System.out.println("To get information about available commands write @help");
+        System.out.println("To start the program from the beginning write @start to exit the program write @exit");
+        FilesEditor();
+    }
+
+    public void FilesEditor() {
+        System.out.println();
+        System.out.println("Write command pleas!");
+        inputFromConsole = new Scanner(System.in).nextLine();
+        if (inputFromConsole.equalsIgnoreCase("create")) {
+            System.out.println("Write path file!to create the file");
+            createFile();
+            System.out.println();
+            FilesEditor();
+        } else if (inputFromConsole.equalsIgnoreCase("read")) {
+            readFile();
+            System.out.println();
+            FilesEditor();
+        } else if (inputFromConsole.equalsIgnoreCase("info")) {
+            infoAboutFile();
+            System.out.println();
+            FilesEditor();
+        } else if (inputFromConsole.equalsIgnoreCase("help")) {
+            help();
+            System.out.println();
+            FilesEditor();
+        } else if (inputFromConsole.equalsIgnoreCase("start")) {
+            System.out.println();
+            FilesEditor();
+        } else if (inputFromConsole.equalsIgnoreCase("exit")) {
+            System.exit(0);
+        } else {
+            tempTextToFile = inputFromConsole;
+            saveText();
+        }
+    }
+
+    private void saveText() {
```

Comment:
this method it's too complicated. Try to simplify that by removing some logic to separate method and you'll get rid of repeating parts